builtins/amp-img.js

```diff
@@ -64,17 +64,12 @@ export class AmpImg extends BaseElement {
   }
 
   /** @override */
-  prerenderAllowed() {
-    return true;
-  }
-
-  /** @override */
   isRelayoutNeeded() {
     return true;
   }
 
   /** @override */
-  layoutCallback() {
+  prerenderCallback() {
```

Comment:
Ah interesting. That's a good note on the changes about "element is laid out" state on prerender. I'll keep an eye on this and see where does this might affect our resources scheduling logic. Let me know if you can think of some places where this might be critical.